forage-android/src/main/java/com/joinforage/forage/android/ForageSDKApi.kt

```diff
@@ -11,7 +11,8 @@ internal interface ForageSDKApi {
     suspend fun tokenizeEBTCard(
         merchantAccount: String,
         bearerToken: String,
-        customerId: String
+        customerId: String,
```

Comment:
@djoksimo clarified that making `customerId` optional is already a known concern for Android and iOS. I'll make this comment as resolved then since that's out of scope for this PR.